waku/standards/core/17/rln-relay.md

```diff
@@ -12,113 +12,149 @@ contributors:
 Section [Payload](#payloads) covers the details about the type and encoding of these fields.
 
-The `merkle_root` contains the root of the Merkle tree.
-
-The `epoch` represents the current epoch.
-
-The `nullifier` is an internal nullifier acting as a fingerprint that allows specifying whether two messages are published by the same peer during the same `epoch`.
-The `nullifier` is a deterministic value derived from `sk` and `epoch` therefore any two messages issued by the same peer (i.e., using the same `sk`) for the same `epoch` are guaranteed to have identical `nullifier`s.
-
-The `share_x` and `share_y` can be seen as partial disclosure of peer's `sk` for the intended `epoch`.
-They are derived deterministically from peer's `sk` and current `epoch` using [Shamir secret sharing scheme](../../../../vac/32/rln-v1.md). 
-If a peer discloses more than one such pair (`share_x`, `share_y`) for the same `epoch`, it would allow full disclosure of its  `sk` and hence get access to its staked fund in the membership contract.
-
-
-The `proof` field is a zero-knowledge proof signifying that: 
-1. The message owner is the current member of the group i.e., her/his identity commitment key `pk` is part of the membership group Merkle tree with the root `merkle_root`.
-2. `share_x` and `share_y`  are correctly computed.
+- The `merkle_root` contains the root of the Merkle tree.
+- The `epoch` represents the current epoch.
+- The `nullifier` is an internal nullifier acting as a fingerprint that allows specifying whether two messages are published by the same peer during the same `epoch`.
+- The `nullifier` is a deterministic value derived from `sk` and
+`epoch` therefore any two messages issued by the same peer
+(i.e., using the same `sk`) for the same `epoch` are guaranteed to have identical `nullifier`s.
+- The `share_x` and `share_y` can be seen as partial disclosure of peer's `sk` for the intended `epoch`.
+They are derived deterministically from peer's `sk` and
+current `epoch` using [Shamir secret sharing scheme](../../../../vac/32/rln-v1.md). 
+
+If a peer discloses more than one such pair (`share_x`, `share_y`) for the same `epoch`, 
+it would allow full disclosure of its  `sk` and 
+hence get access to its staked fund in the membership contract.
+
+- The `proof` field is a zero-knowledge proof signifying that: 
+
+1. The message owner is the current member of the group i.e.,
+the peer's identity commitment key, `pk`,
+is part of the membership group Merkle tree with the root `merkle_root`.
+2. `share_x` and `share_y` are correctly computed.
 3. The `nullifier` is constructed correctly.
 For more details about the proof generation check [RLN](../../../../vac/32/rln-v1.md)
 The proof generation relies on the knowledge of two pieces of private information i.e., `sk` and `authPath`.
 The `authPath` is a subset of Merkle tree nodes by which a peer can prove the inclusion of its `pk` in the group. <!-- TODO refer to RLN RFC for authPath def -->
```

Comment:
Should `authPath` be replaced with `path_elements` and `indices`?

Reply:
yes, it should